test/cmd/basicresources.sh

```diff
@@ -307,6 +307,11 @@ os::cmd::expect_success 'oc new-app node'
 os::cmd::expect_success_and_text 'oc get all' 'is/test1'
 os::cmd::expect_success_and_not_text 'oc get is' 'is/test1'
 echo "resource printer: ok"
+# Test that resource printer includes namespaces for buildconfigs with custom strategies
+os::cmd::expect_success 'oc create -f examples/sample-app/application-template-custombuild.json'
+os::cmd::expect_failure_and_text 'oc new-app ruby-helloworld-sample' 'deploymentconfig "frontend" created'
+os::cmd::expect_success_and_text 'oc get all --all-namespaces' 'cmd-basicresources[\ ]+bc\/ruby\-sample\-build'
+echo "resource printer namespace handling: ok"
```

Comment:
Instead of adding new tests to this suite, let's break out resource printer tests from `basicresources`